### PR TITLE
Set payment description

### PR DESCRIPTION
**Pull request by malinajirka**
Parent issue #4039 

Merge instrucitons
1. Make sure https://github.com/woocommerce/woocommerce-android/pull/4106 is merged
2. Remove "do not merge" label
3. Merge this PR

Sets payment description to each card present payment. The description is visible in Stripe's dashboard next to the transaction.

To test
- use a store with WCPay in test mode and orders in US $
1. Open detail of an order
2. Tap on "Collect payment"
3. Walk through the connection and payment flows
4. Open Stripe's dashboard (if you don't have access to the dashboard, just put a breakpoint [here](https://github.com/woocommerce/woocommerce-android/blob/c696819e9a7b74b09dc8e8871d79a58c9d3bf177/libs/cardreader/src/debug/java/com/woocommerce/android/cardreader/internal/payments/actions/CreatePaymentAction.kt#L59) and verify the description is set)
5. Verify the transaction has description set

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/cardreader/CardReaderPaymentViewModel.kt [hunk 1]
```diff
@@ -197,6 +204,13 @@ class CardReaderPaymentViewModel @Inject constructor(
     private suspend fun loadOrderFromDB() =
         withContext(dispatchers.io) { orderStore.getOrderByIdentifier(arguments.orderIdentifier) }
 
+    private fun getPaymentDescription(order: WCOrderModel): String =
```
Comment (kidinov) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/cardreader/CardReaderPaymentViewModel.kt]: Matter of taste probably, but I would do it as an extension function. We kind of taking description from order in this case (at least as I see this =) )

File: libs/cardreader/src/main/java/com/woocommerce/android/cardreader/CardReaderManager.kt [hunk 2]
```diff
@@ -16,7 +16,9 @@ interface CardReaderManager {
     suspend fun connectToReader(cardReader: CardReader): Boolean
 
     // TODO cardreader Stripe accepts only Int, is that ok?
+    // TODO cardreader wrap payment params with a data class
```
Comment (kidinov) [inline libs/cardreader/src/main/java/com/woocommerce/android/cardreader/CardReaderManager.kt:20]: 👍

**Review by kidinov — APPROVED** [review]
👍

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/cardreader/CardReaderPaymentViewModel.kt (hunk 1 above)
Comment (malinajirka) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/cardreader/CardReaderPaymentViewModel.kt]: Update in 391f020fab5b32df84b75955621b54dd486b6df9 ;)

**Pull request merged**